config/liota.conf

```diff
@@ -17,11 +17,11 @@ collect_thread_pool_size = 30
 
 [PKG_CFG]
 pkg_path = /etc/liota/packages
-pkg_msg_pipe = /var/tmp/liota/package_messenger.fifo
+pkg_msg_pipe = /etc/liota/packages/package_messenger.fifo
```

Comment:
Currently, for package manager and device discovery related Named Pipe, they are restricted to be only accessible by the owner.